pkgs/top-level/all-packages.nix

```
@@ -31428,6 +31428,8 @@ with pkgs;
 
   cups-bjnp = callPackage ../misc/cups/drivers/cups-bjnp { };
 
+  cups-brother-dcp9020cdw = pkgsi686Linux.callPackage ../misc/cups/drivers/brother-dcp9020cdw { };
```

As we already have a [`pkgs/misc/cups/drivers/brother/`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/misc/cups/drivers/brother) folder, will be nice to create a `dcp9020cdw/` folder in it and put the `default.nix` file inside. The path here will also needs update otherwise we will not able to build this, you may test by running `nix-build -A cups-brother-dcp9020cdw` at the root of the repository.

```suggestion
  cups-brother-dcp9020cdw = pkgsi686Linux.callPackage ../misc/cups/drivers/brother/dcp9020cdw { };
```

---

Ok, I'll try to change that.